build.gradle

```diff
@@ -199,7 +199,6 @@ task goIntegrationTests() {
 
 task pythonPreCommit() {
   dependsOn ":sdks:python:test-suites:tox:pycommon:preCommitPyCommon"
-  dependsOn ":sdks:python:test-suites:tox:py35:preCommitPy35"
```

Comment:
FYI @tvalentyn I just merged this PR including a commit to drop python 3.5 in the precommit